.gitlab-ci.yml

```diff
@@ -0,0 +1,215 @@
+# Copyright 2018 Red Hat, Inc.
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# Part of pacemaker project
+# SPDX-License-Identifier: FSFAP
+
+variables:
+  GIT_DEPTH: "16"
```

Comment:
It's a reminder to eventually implement nice-to-have automated
bisection on failure, and 16 commits back seems to be a reasonable
limit given the past track and the timing math (in worst case and
serial handling, it would grow beyond a default total limit of
1 hour but not that much), and won't bring much burden either,
while substantially better than default of whole history.

ISTR that this number should not jump across jobs per documentation,
hence set globally, with this anticipation for future.